src/doppler/sinks/syslogwriter/https_writer_test.go

```diff
@@ -98,15 +99,38 @@ var _ = Describe("HttpsWriter", func() {
 
 		})
 
+		Context("when a request times out", func() {
+			BeforeEach(func() {
+				requestChan <- []byte{}
```

Comment:
Is this send used only to make the ServeHTTP block? It seems like it's pretty brittle considering that it depends entirely on the fact that the channel is buffered with size=1. Maybe a better approach would be to use a different handler that allows for parameterized sleep times. So we could validate that the timeout does not occur when the sleep time is less than the timeout, and does occur if the it is greater.